unicodetools/data/ucd/dev/UnicodeData.txt

```diff
@@ -21870,6 +21870,60 @@ FFFD;REPLACEMENT CHARACTER;So;0;ON;;;;;N;;;;;
 11DA7;GUNJALA GONDI DIGIT SEVEN;Nd;0;L;;7;7;7;N;;;;;
 11DA8;GUNJALA GONDI DIGIT EIGHT;Nd;0;L;;8;8;8;N;;;;;
 11DA9;GUNJALA GONDI DIGIT NINE;Nd;0;L;;9;9;9;N;;;;;
+11DB0;TOLONG SIKI LETTER I;Lo;0;L;;;;;N;;;;;
+11DB1;TOLONG SIKI LETTER E;Lo;0;L;;;;;N;;;;;
+11DB2;TOLONG SIKI LETTER U;Lo;0;L;;;;;N;;;;;
+11DB3;TOLONG SIKI LETTER O;Lo;0;L;;;;;N;;;;;
+11DB4;TOLONG SIKI LETTER A;Lo;0;L;;;;;N;;;;;
+11DB5;TOLONG SIKI LETTER AA;Lo;0;L;;;;;N;;;;;
+11DB6;TOLONG SIKI LETTER P;Lo;0;L;;;;;N;;;;;
+11DB7;TOLONG SIKI LETTER PH;Lo;0;L;;;;;N;;;;;
+11DB8;TOLONG SIKI LETTER B;Lo;0;L;;;;;N;;;;;
+11DB9;TOLONG SIKI LETTER BH;Lo;0;L;;;;;N;;;;;
+11DBA;TOLONG SIKI LETTER M;Lo;0;L;;;;;N;;;;;
+11DBB;TOLONG SIKI LETTER T;Lo;0;L;;;;;N;;;;;
+11DBC;TOLONG SIKI LETTER TH;Lo;0;L;;;;;N;;;;;
+11DBD;TOLONG SIKI LETTER D;Lo;0;L;;;;;N;;;;;
+11DBE;TOLONG SIKI LETTER DH;Lo;0;L;;;;;N;;;;;
+11DBF;TOLONG SIKI LETTER N;Lo;0;L;;;;;N;;;;;
+11DC0;TOLONG SIKI LETTER TT;Lo;0;L;;;;;N;;;;;
+11DC1;TOLONG SIKI LETTER TTH;Lo;0;L;;;;;N;;;;;
+11DC2;TOLONG SIKI LETTER DD;Lo;0;L;;;;;N;;;;;
+11DC3;TOLONG SIKI LETTER DDH;Lo;0;L;;;;;N;;;;;
+11DC4;TOLONG SIKI LETTER NN;Lo;0;L;;;;;N;;;;;
+11DC5;TOLONG SIKI LETTER CA;Lo;0;L;;;;;N;;;;;
+11DC6;TOLONG SIKI LETTER CH;Lo;0;L;;;;;N;;;;;
+11DC7;TOLONG SIKI LETTER J;Lo;0;L;;;;;N;;;;;
+11DC8;TOLONG SIKI LETTER JH;Lo;0;L;;;;;N;;;;;
+11DC9;TOLONG SIKI LETTER NY;Lo;0;L;;;;;N;;;;;
+11DCA;TOLONG SIKI LETTER K;Lo;0;L;;;;;N;;;;;
+11DCB;TOLONG SIKI LETTER KH;Lo;0;L;;;;;N;;;;;
+11DCC;TOLONG SIKI LETTER G;Lo;0;L;;;;;N;;;;;
+11DCD;TOLONG SIKI LETTER GH;Lo;0;L;;;;;N;;;;;
+11DCE;TOLONG SIKI LETTER NG;Lo;0;L;;;;;N;;;;;
+11DCF;TOLONG SIKI LETTER Y;Lo;0;L;;;;;N;;;;;
+11DD0;TOLONG SIKI LETTER R;Lo;0;L;;;;;N;;;;;
+11DD1;TOLONG SIKI LETTER L;Lo;0;L;;;;;N;;;;;
+11DD2;TOLONG SIKI LETTER V;Lo;0;L;;;;;N;;;;;
+11DD3;TOLONG SIKI LETTER NNY;Lo;0;L;;;;;N;;;;;
+11DD4;TOLONG SIKI LETTER S;Lo;0;L;;;;;N;;;;;
+11DD5;TOLONG SIKI LETTER H;Lo;0;L;;;;;N;;;;;
+11DD6;TOLONG SIKI LETTER X;Lo;0;L;;;;;N;;;;;
+11DD7;TOLONG SIKI LETTER RR;Lo;0;L;;;;;N;;;;;
+11DD8;TOLONG SIKI LETTER RRH;Lo;0;L;;;;;N;;;;;
+11DD9;TOLONG SIKI SIGN SELA;Lo;0;L;;;;;N;;;;;
```

Comment:
This is a vowel length mark, see p. 5 and the chart on p. 13 of the proposal L2/23-024.

I think this should be gc=Lm, and in PropList.txt, should have the Diacritic and Extender properties (like U+02D0).

The proposal bundles U+11DDA with it. For that noe I don’t know whether it is more appropriate to treat it as a letter that is a glottal stop or as modifier that does mysterious syllable coda things. @Ken-Whistler, what do you think?

Reply:
I don't think it (hecakā) is a vowel length mark, as contrasted with selā, which is. The glottal stop seems to be a full segment, although perhaps in restricted distribution. See the discussion of Kurukh and Brahui phonology on Wikipedia, for instance, which shows length distinctions (and in some cases nasalization distinctions) for vowels, which when marked in the orthography can be considered vowel modifiers and/or extenders, etc. But I think it is safest to just leave the glottal stop as Lo and treat it as another letter.

Reply:
Does this change also require changes to LineBreak.txt?